Exercise_C.js

```diff
@@ -12,7 +12,18 @@ If the number is divisible by 3 and 5 it should log Hong Kong.
 
 See README.md if you need help running your code.
 */
-
-for (let i = 1; i <= 30; i++) {
-  // CODE HERE
+function hongkong() {
+  for (let i = 1; i <= 30; i++) {
+    if(i % 3 == 0 && i % 5 == 0) {
+      console.log('Hong Kong');  
+    } else if (i % 5 == 0) {
+      console.log('Kong');  
+    } else if(i % 3 == 0) {
+      console.log('Hong');  
+    } else {
+      console.log(i);        
+    }
+  }
 }
+
+console.log(hongkong());

```

Comment:
if you're using function then you'll need to return the answer as well after console logging